.eslintrc

```diff
@@ -262,4 +266,4 @@ rules:
   valid-typeof: [2]
   wrap-iife: [2, inside]
   yield-star-spacing: [2, {before: true, after: false}]
-  yoda: [2, never]
+  yoda: [0]
```

Comment:
It's in the code... I didn't feel like modifying any code, just the linting... I'll work on it too!